cmd/sync-tags/main.go

```diff
@@ -193,6 +206,34 @@ func main() {
 			continue
 		}
 
+		// update Godeps.json to point to actual tagged version in the dependencies. This version might differ
+		// from the one currently in Godeps.json because the other repo could have gotten more commit for this
+		// tag, but this repo didn't.
```

Comment:
Add a link to https://github.com/kubernetes/publishing-bot/issues/12? The issue explains the problem with more details.

Reply:
👍